images/php/fpm/Dockerfile

```
@@ -64,11 +64,16 @@ RUN apk add --no-cache fcgi \
         # for webp
         libwebp-dev \
         postgresql-dev \
+	# for imagemagick
+	imagemagick \
+	imagemagick-libs \
```

```suggestion
        # for imagemagick
        imagemagick \
        imagemagick-libs \
```

I would indent this to be on the same level as above.